### PR TITLE
add Calcit

### PR DESCRIPTION
**Pull request by jiyinyiyong**
Home page for Calcit http://calcit-lang.org/ .
Crate https://crates.io/crates/calcit_runner .
Demo of a Calcit-js app https://github.com/calcit-lang/editor .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +213,7 @@ presented separately here.
 
 <!-- 75 -->
 [Parcel JavaScript Transformer]: https://github.com/parcel-bundler/parcel
+[Calcit]: https://github.com/calcit-lang/calcit_runner.rs
```
Comment (jiyinyiyong) [inline README.md:216]: unstable grouping method...?

**Comment by jiyinyiyong** [comment]
I found the numbers of stars are outdated now, needs an update?

**Comment by jiyinyiyong** [comment]
hello? @alilleybrinker

File: README.md (hunk 1 above)
Comment (alilleybrinker) [reply README.md:216]: If you have an idea for changing the grouping method, go ahead and suggest one. 👍🏻

**Comment by alilleybrinker** [comment]
Thanks for the ping. Looks fine to me. As for the stars, everything currently shown was only checked when added to the repo. If you want to go through and update them, that'd be helpful!

**Pull request merged**